app/experimenter/experiments/api/v6/serializers.py

```diff
@@ -77,7 +75,7 @@ def get_features(self, obj):
         for fv in obj.feature_values.all():
             feature_value = {
                 "featureId": fv.feature_config and fv.feature_config.slug or "",
-                "enabled": fv.enabled,
+                "enabled": True,
```

Comment:
same as here, or may be add a constant and add a note where the constant is declared 🤔